org-roam.el

```diff
@@ -623,8 +623,9 @@ If FILE-PATH is nil, use the current file."
   (let (result)
       ;; We need to handle the special case of the file property drawer (at outline level 0)
       (org-with-point-at (point-min)
-        (when-let ((id (org-entry-get nil "ID")))
-           (push (vector id file-path (org-outline-level)) result)))
+        (when-let ((id (org-entry-get nil "ID"))
+                   (before-first-heading (= 0 (org-outline-level))))
```

Comment:
Might want to swap the order of checks, I think the outline level check is cheaper.

Reply:
True that, updated.